Library/Homebrew/cmd/install-vendor.sh

```diff
@@ -0,0 +1,108 @@
+source "$HOMEBREW_LIBRARY/Homebrew/utils/lock.sh"
+
+if [[ -n "$HOMEBREW_OSX" ]]
+then
+  Git_URL=""
```

Comment:
FYI, I'm just choosing the file structure which is similar to what Apple does in `/System/Library/Frameworks/Ruby.framework/Versions/`